### PR TITLE
adding page with directions for editing code in sublime and vim

### PR DESCRIPTION
**Pull request by ghost**
http://screencast.com/t/AS8phD6Vcv

### CODE REVIEW

**Comment by jtgraphic** [comment]
logos could be actual size displayed in page but nbd. :cake:

File: views/layouts/default.lavender [hunk 1]
```diff
@@ -104,6 +104,11 @@ body
               url: '/about.html',
               target: '_self'
             },
+            {
+            title: 'Editing',
```
Comment (mowchan) [inline views/layouts/default.lavender]: Indentation

File: views/syntax.lavender [hunk 2]
```diff
@@ -0,0 +1,59 @@
+
+extends layouts/default
+
+-current_page = "Syntax"
+
+block title
+	| Syntax View - Lavender Templating Engine for PHP
+
+block content
+	.container
```
Comment (mowchan) [inline views/syntax.lavender]: Indentation

File: views/syntax.lavender [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      .col-md-8.col-md-offset-2.about.img-rounded
+        .col-md-5
+          img.img-responsive.center-block(src='/img/logos/sublime.png')          
+        .col-md-4.col-md-offset-2
+          h1 Sublime
+          p
+            | Since Lavender is based off of Jade, any text editor that can show Jade syntax will work.
+            | We prefer 
+            a(href='http://www.sublimetext.com/', target='_blank') <b>Sublime</b>
+
```
Comment (mowchan) [inline views/syntax.lavender]: Whitespace

**Comment by mowchan** [comment]
Aside from those 3 text formatting issues :cake:

File: views/syntax.lavender [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+extends layouts/default
+
+-current_page = "Syntax"
+
+block title
+	| Syntax View - Lavender Templating Engine for PHP
```
Comment (mowchan) [inline views/syntax.lavender]: This is indented pretty far also

File: views/syntax.lavender [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+            a(href='http://www.sublimetext.com/', target='_blank') <b>Sublime</b>
+            | but any editor that can read Jade will do.
+  .container
+    .row
+      .col-md-8.col-md-offset-2.about.img-rounded
+          h2 How it's done
+          p
+            | The first step is to install package control in Sublime
+            | follow the instructions for your preferred version of Sublime at
+            a(href='https://packagecontrol.io/installation', target='_blank') <b>Packagecontrol.io</b>
```
Comment (mowchan) [inline views/syntax.lavender]: `<b>` tags (along with `<i>` tags) are a thing of the past. You should use `<strong>` for bold and `<em>` for italics. The purpose for the new tags is so you can have your own definition of "emphasis"... but out of the box they do the same thing as the old tags.

Comment (mowchan) [inline views/syntax.lavender]: Also, I wouldn't put it on the same line like that. I'd go to a new line and indent, then use normal lavender syntax.

File: views/syntax.lavender [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+          p
+            | Next, press cmd + shift + p, type install, hit enter
+            | type jade, hit enter.  Jade will now be an option in the syntax list.
+            | Open a <b>.lavender</b> file, go to
```
Comment (mowchan) [inline views/syntax.lavender]: Same thing here. Do new nested tags like this: 

```
p Start your sentence
  strong BOLD
  | continue your sentence
```

File: views/syntax.lavender [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+  .container               
+    .row
+      .col-md-8.col-md-offset-2.about.img-rounded
+        .col-md-5
+          img.img-responsive.center-block(src='/img/logos/vim.png')          
+        .col-md-4.col-md-offset-2
+          h1 Vim
+          p
+            | Do you prefer 
+            a(href='http://www.vim.org/download.php', target='_blank' ) <b>Vim?</b>
```
Comment (mowchan) [inline views/syntax.lavender]: Same

File: views/syntax.lavender [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+            | Get the 
+            a(href='https://github.com/digitaltoad/vim-jade', target='_blank') <b>digitaltoad package</b>
```
Comment (mowchan) [inline views/syntax.lavender]: Same

File: views/syntax.lavender [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+    .row
+      .col-md-8.col-md-offset-2.about.img-rounded
+        .col-md-5
+          img.img-responsive.center-block(src='/img/logos/sublime.png')          
+        .col-md-4.col-md-offset-2
+          h1 Sublime
+          p
+            | Since Lavender is based off of Jade, any text editor that can show Jade syntax will work.
+            | We prefer 
+            a(href='http://www.sublimetext.com/', target='_blank') <b>Sublime</b>
```
Comment (mowchan) [inline views/syntax.lavender]: Same

**Comment by mowchan** [comment]
Make that last change and you're good! :cake:

**Pull request merged**